gpu/tasks/nvidia_ctoolkit.yml

```diff
@@ -0,0 +1,44 @@
+---
+- name: Install NVIDIA Container Toolkit and Runtime
+  ansible.builtin.apt:
+    name: 
+      - nvidia-container-toolkit
+      - nvidia-container-runtime
+    state: present
+    update_cache: yes
+
+- name: Check if NVIDIA runtime configuration file exists
+  ansible.builtin.stat:
+    path: /etc/nvidia-container-runtime/config.toml
+  register: nvidia_config_file
+
+- name: Enable NVIDIA visible devices as volume mounts
+  ansible.builtin.lineinfile:
+    path: /etc/nvidia-container-runtime/config.toml
+    regexp: '#accept-nvidia-visible-devices-as-volume-mounts = false'
+    line: 'accept-nvidia-visible-devices-as-volume-mounts = true'
+    backrefs: yes
+  when: nvidia_config_file.stat.exists
+
+- name: Disable NVIDIA visible devices envvar when unprivileged
+  ansible.builtin.lineinfile:
+    path: /etc/nvidia-container-runtime/config.toml
+    regexp: '#accept-nvidia-visible-devices-envvar-when-unprivileged = true'
+    line: 'accept-nvidia-visible-devices-envvar-when-unprivileged = false'
```

Comment:
could you please also add a separate task that actually checks the `accept-nvidia-visible-devices-envvar-when-unprivileged = false` parameter is present (and not commented) in the `config.toml`?

Because the layout might change so we would learn quickly that `regexp` isn't working anymore should they change it.